<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_survival">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>140</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Survival Analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_drug">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Drug Analysis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_CRISPR">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>140</y>
      <width>180</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>CRISPR Analysis</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="icon_survival">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="icon_drug">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>50</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="icon_CRISPR">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>50</y>
      <width>91</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_survival">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>30</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_drug">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>151</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_CRISPR">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>171</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="btn_cellline">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cell Line</string>
    </property>
   </widget>
   <zorder>label_survival</zorder>
   <zorder>label_drug</zorder>
   <zorder>label_CRISPR</zorder>
   <zorder>icon_survival</zorder>
   <zorder>icon_drug</zorder>
   <zorder>icon_CRISPR</zorder>
   <zorder>btn_drug</zorder>
   <zorder>btn_survival</zorder>
   <zorder>btn_CRISPR</zorder>
   <zorder>btn_cellline</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCreate"/>
   <addaction name="menuHelp"/>
   <addaction name="menuExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionNew_2">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
